### DISCUSSION TITLE
Where to hire FastAPI developers?

### DISCUSSION THREAD

**Discussion by maxcorbeau** [fastapi/fastapi#5094] [Questions] [answered]
**Background**: FastAPI has its own way of doing things (https://fastapi.tiangolo.com/alternatives/) and from experience a bit of a learning curve. I'd like to hire developers to help me with some FastAPI projects however all I find are django/flask devs.

**Question**: What's the best way to hire developers with strong experience in FastAPI? Can I post job details here? Should I post on https://dev.to/t/fastapi? I'm afraid of posting in the wrong place and give the feeling I'm hijacking a medium for personal business. I truly want to find great FastAPI experts, but the right way :)

**Comment by acidjunk** [comment]
What kind of projects do you have in mind? 

Most people that are good with Flask and modern Python will find it easy, and fun, to work with FastAPI. At SURF we switched a year ago from Flask to FastAPI; and we didn't have big problems. We could re-use a lot of our existing python code. The async stuff can be tricky though. But indeed, every framework has its pitfalls.

I can imagine that reddit is also a good source. Or recruitment agency's.

**Reply by maxcorbeau** [reply]
Hi @[acidjunk](https://github.com/acidjunk): projects vary but all revolved around creating a base project in FastAPI which I later on extend myself. Right now I'm looking for someone to create a FastAPI + Google Oauth 2 template: [requirements](https://docs.google.com/presentation/d/1W_o7sIrMPrcuC_r1r8ja3sWADNs_WGwn6pk9hA1mGdM/)

**Comment by tiangolo** [comment, marked as answer]
You can probably post on the Discord chat, there's a job board: https://fastapi.tiangolo.com/help-fastapi/#join-the-chat
